<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="DESKTOP-PH7A22V" failures="0" tests="2" name="com.testclass.saucedemo.TS_yourinfo_continue_function" time="14.150" errors="1" timestamp="2022-04-14T08:55:42 IST" skipped="0">
  <testcase classname="com.testclass.saucedemo.TS_yourinfo_continue_function" name="closebrowser" time="10.382">
    <error message="Method com.testclass.saucedemo.BasetestClass.closebrowser() didn&#039;t finish within the time-out 5000" type="org.testng.internal.thread.ThreadTimeoutException">
      <![CDATA[org.testng.internal.thread.ThreadTimeoutException: Method com.testclass.saucedemo.BasetestClass.closebrowser() didn't finish within the time-out 5000
at org.testng.internal.MethodInvocationHelper.invokeWithTimeoutWithNoExecutor(MethodInvocationHelper.java:348)
at org.testng.internal.MethodInvocationHelper.invokeWithTimeout(MethodInvocationHelper.java:281)
at org.testng.internal.MethodInvocationHelper.invokeWithTimeout(MethodInvocationHelper.java:266)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:64)
at org.testng.internal.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:385)
at org.testng.internal.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:321)
at org.testng.internal.TestMethodWorker.invokeAfterClassMethods(TestMethodWorker.java:217)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:130)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
]]>
    </error>
  </testcase> <!-- closebrowser -->
  <system-out/>
  <testcase classname="com.testclass.saucedemo.TS_yourinfo_continue_function" name="continuefunctionality" time="3.768"/>
  <system-out/>
</testsuite> <!-- com.testclass.saucedemo.TS_yourinfo_continue_function -->
